### ISSUE TITLE
u_tau and tau_wall don't have to be arrays

### ISSUE THREAD

**Issue by Niceno** [DelNov/T-Flows#132]
Dear guys,

I was working in implementation of multi-domain (and later multi-material) capability and have noticed that two arrays; u_tau and tau_wall don't have to be arrays at all.  They are used in four functions (`Src_Kin_K_Eps`, `Src_Eps_K_Eps`, `Src_Kin_K_Eps_Zeta_F` and `Src_Eps_K_Eps_Zeta_F`) but are always re-computed, never passed from one to another.

I suggest we turn them into normal variables defined locally in each of these functions.  Is this OK for everyone?

**Comment by Muhammmedaly** [comment]
I agree with that it shouldn’t be a “global” array but still be a local one browsing through the boundary cells (since tau_wall can differ from one boundary cell to another in irregular geometries). What do you all think ?

**Comment by Niceno** [comment]
I think local variable would be sufficient since it doesn't store values in boundary cells, rather in cell inside the domain.
